Gemfile

```diff
@@ -34,6 +34,7 @@ group :development do
   gem 'listen', '~> 3.3'
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
   gem 'spring'
+  gem 'annotate'
```

Comment:
You can add the version you installed, like the one 3 lines up: `gem 'listen', '~> 3.3'`

Reply:
In this case it would be `3.1.1` (you can check your gemfile.lock)

Reply:
Nice, and why doesn't spring have the version? Is that intended?

Reply:
That's a really good question 😆 . I don't know, in the meantime, feel free to merge this PR without changing `gem 'spring'`

Reply:
ok great 😄